seqr/views/apis/data_manager_api_tests.py

```diff
@@ -888,7 +888,7 @@ def test_load_phenotype_prioritization_data(self, mock_logger, mock_subprocess):
         mock_logger.info.assert_called_with('create PhenotypePrioritizations', self.data_manager_user, db_update=db_update)
         saved_data = _get_json_for_models(PhenotypePrioritization.objects.filter(tool='lirical'),
                                           nested_fields=[{'fields': ('individual', 'guid'), 'key': 'individualGuid'}])
-        self.assertListEqual(saved_data, EXPECTED_LIRICAL_DATA)
+        self.assertListEqual(sorted(saved_data, key=lambda d: d['individualGuid']), EXPECTED_LIRICAL_DATA)
```

Comment:
would be cleaner to add an `order_by` to the above query for the underlying `PhenotypePrioritization` used for `saved_data`